components.tsx

```diff
@@ -263,6 +266,21 @@ function PrettyDate({ date, timezone }: { date: Date; timezone?: string }) {
   return <time dateTime={date.toISOString()}>{formatted}</time>;
 }
 
+function Tags({ tags }: { tags?: string[] }) {
+  return (
+    <section class="flex gap-x-2 flex-wrap">
```

Comment:
Won't this cause to render empty space if no tags are present? I suggest to only render this component if tags are present

Reply:
Okay, I will fix this